newsletter.md

```diff
@@ -0,0 +1,14 @@
+
+During the past few months, the team has been working hard to make new features available in the OpenPaaS MVP release.
+
+
+The team implemented JMAP sharing, allowing one user to share a folder with another user of the same domain. The sharee gets the right, if allowed, to read, mark, delete and moves emails. This is especially useful to delegate the management of your emails to someone else. You can see a quick [video](https://www.youtube.com/watch?v=iKygmVKH-xU) presenting JMAP sharing feature in INBOX.
+
+
+We also implemented drafts. The email you compose can now be saved in INBOX, reopened, re-edited. Moving them to the Outbox mailbox will actually trigger the sending. You can see a quick [video](https://www.youtube.com/watch?v=iKygmVKH-xU) presenting the draft feature in INBOX.
+
+
+After implementing these features, we were also involved in the "polish" sprint (see above?), allowing us to fix some little but annoying issues.
```

Comment:
``(see above?)``
is it meant to stay like that ?

Reply:
@strokyl : i think @chibenwa was wondering if a more global part in this newsletter was going to present the polish sprint to which the entire OP team participated. And the answer is "yes" so i will handle this "see above" mention.